src/tox_lsr/test_scripts/runqemu.py

```diff
@@ -57,6 +57,11 @@
     "https://pagure.io/fork/rmeggins/standard-test-roles/raw/"
     "linux-system-roles/f/inventory/standard-inventory-qcow2"
 )
+DEFAULT_LSR_QEMU_REPORT_ERRORS_URL = (
+    "https://raw.githubusercontent.com/linux-system-roles/auto-maintenance/"
+    "0cf66855f269bc7f3d726b2e1e4c6e5e8124cc63/callback_plugins/"
+    "lsr_report_errors.py"
+)
```

Comment:
Hardcoded SHA is a maintenance burden. This should be `main` or at least a stable tag (in case it's not meant to be a stable API).

Reply:
Yes.  This will point to `main` once https://github.com/linux-system-roles/auto-maintenance/pull/376 is merged.